### DISCUSSION TITLE
Upgrade from 5.2.8 to 5.8.1

### DISCUSSION THREAD

**Discussion by prateeksisodia** [concourse/concourse#6114] [Help & Support]
The issue with high memory utilization was solved as part of v5.2.8 and 5.5.x
https://github.com/concourse/concourse/issues/5341

Currently our concourse is on version 5.2.8 and we are planning an upgrade to 5.8.1, is this issue of high memory consumption not there is version 5.8.1? is it possible to use the flag CONCOURSE_DISABLE_LIST_ALL_JOBS in v 5.8.1?

Also, would like to get you inputs on if we should upgrade to v5.5.x or 5.8.1.
We are not ready for upgrade to v6.x yet.

**Comment by jamieklassen** [comment]
I couldn't remember which lines we put that patch into. I ran
```
$ docker run --rm -it concourse/concourse:5.5.11 web --help | grep disable-list-all-jobs
          --disable-list-all-jobs
```
so it's definitely in 5.5.11, but
```
$ docker run --rm -it concourse/concourse:5.8.1 web --help | grep disable-list-all-jobs
```
shows that it's not in 5.8.1. It was a weird situation where we built that feature pretty explicitly against the 5.5.x codebase since that was a "long-term supported" version.
